trillian.proto

```diff
@@ -27,6 +27,22 @@ import "google/protobuf/any.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
 
+
+// LogSignatureFormat specifies the fields that are covered by the
+// SignedLogRoot signature, as well as their ordering and formats. 
+enum LogSignatureFormat {
+   LOG_SIG_FORMAT_UNKNOWN = 0;
+   LOG_SIG_FORMAT_V1 = 1;
```

Comment:
Could we add a pointer indicating where the signature format / ordering for `FORMAT_V1` is defined/documented?